releases/v1.3.0

```diff
@@ -0,0 +1,43 @@
+Highlights:
+* Improved Kotlin support (#319)
```

Comment:
Can you add explicit information that "Kotlin users have to include temporal-kotlin module in the classpath"?